lib/Dialect/FIRRTL/Transforms/CheckInit.cpp

```diff
@@ -0,0 +1,189 @@
+//===- CheckInit.cpp - Ensure all wires are initialized ------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the CheckInit pass.  This pass checks that all wires are
+// initialized (connected too).
```

Comment:
```suggestion
// initialized (connected to).
```

Description says wires and ports.  Implementation suggests checks more than those.  Not sure how to generalize without enumerating, but maybe that's the way to go.

Reply:
You have to track all assignable things, but most of them are effectively ignored.